src/libraries/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -248,6 +248,7 @@
     <Compile Include="System\UriIdnScope.cs" />
     <Compile Include="$(CommonPath)System\IO\TempFileCollection.cs"
              Link="Common\System\IO\TempFileCollection.cs" />
+    <Compile Include="$(CommonPath)System\Obsoletions.cs" Link="Common\System\Obsoletions.cs" />
```

Comment:
I can't find if you are actually using this as most of the places I found are using the resx string instead, is this actually needed?

Reply:
Yup. The [Obsolete(...)] attribute added to `SettingsSerializeAs.Binary` uses it